src/main/java/com/epam/preprod/sirenko/Demo.java

```diff
@@ -1,11 +1,22 @@
 package com.epam.preprod.sirenko;
 
 import com.epam.preprod.sirenko.containers.CommandContainer;
+import com.epam.preprod.sirenko.services.impl.SerializationServiceImpl;
 import com.epam.preprod.sirenko.util.ConsoleReader;
 
 import java.io.*;
 
 public class Demo {
+	private SerializationServiceImpl serializationServiceImpl;
```

Comment:
Kolesnikov told us not to do this because in the source code there are no empty lines between class name and field. Checked it again in ArrayList:

public class ArrayList<E> extends AbstractList<E>
        implements List<E>, RandomAccess, Cloneable, java.io.Serializable
{
    private static final long serialVersionUID = 8683452581122892189L;

Maybe it depends on project?